examples/pubsub/tutorial_pubsub_mqtt_publish.c

```diff
@@ -68,12 +71,12 @@
 #endif
 
 // Uncomment the following line to enable MQTT via TLS for the example
-// #define EXAMPLE_USE_MQTT_TLS
+#define EXAMPLE_USE_MQTT_TLS
 
 #ifdef EXAMPLE_USE_MQTT_TLS
 #define USE_TLS_OPTION_NAME             "mqttUseTLS"
 #define MQTT_CA_FILE_PATH_OPTION_NAME   "mqttCaFilePath"
-#define CA_FILE_PATH                    "/path/to/server.cert"
+#define CA_FILE_PATH                    "/mnt/d/Home/Dokumente/Arbeit/Fraunhofer/Mqtt_Cert/mosquitto.org.crt"
```

Comment:
isn't `#define CA_FILE_PATH                    "/path/to/server.cert"`  correct to make sure automated tests work?

Reply:
The automatic tests run the files from the tests folder. However, my absolute path must not appear in the example. I will change this.
Thanks!